src/Onboarding/Wizard/Page.php

```diff
@@ -118,6 +127,13 @@ public function enqueue_scripts() {
 
 		wp_set_script_translations( 'give-admin-onboarding-wizard-app', 'give' );
 
+		$formID           = $this->formRepository->getOrMake();
+		$featureGoal      = get_post_meta( $formID, '_give_goal_option', true );
```

Comment:
These inline `get_post_meta` calls could be extracted to a `FormModel` to fo with `FormRepository`, but I think that is a separate concern which warrants a separate change.

Reply:
Good note, thanks for explaining your thinking here.

Reply:
https://github.com/impress-org/givewp/issues/5048